processor/extract_msg_pack.py

```diff
@@ -30,6 +30,12 @@ def main(bin_fn, dest_fn):
                 trans_wc = - rot_wc * trans_cw
                 output[key][needed_key] = [
                     trans_wc[0, 0], trans_wc[1, 0], trans_wc[2, 0]]
+            elif needed_key == 'rot_cw':
+                    print(key_frames[key]["rot_cw"])
```

Comment:
do we need this file now, looks like you integrated the logic in the above file itself

Reply:
yes, both extract_msg_pack.py and extract_key_frames.py files are not needed shall i remove them

Reply:
yeah you can remove them.